cdip_admin/integrations/models/v2/models.py

```diff
@@ -192,6 +192,20 @@ def _pre_save(self, *args, **kwargs):
             if "broker" not in self.additional:
```

Comment:
Shorthand could be beneficial here too. This eliminates the chance that one of the two literal `"broker"` values is accidentally changed.
```python
    self.additional.setdefault('broker', 'gcp_pubsub')
```

Reply:
great idea, and we can probably remove this setting sometime as the only supported broker is PubSub now